### PR TITLE
Pushing Popup and Sketch changes

### PR DESCRIPTION
**Pull request by YajurG**
## Description
Pushing changes to Popup and Sketch changes upstream

### Why

Resolves [#302, #307]

Note: While the above issues are closed, there was some additional feedback given and changes requested from the Accessibility team, which are addressed in this PR even though the linked issues are resolved.

### What

Changed 'Close Popup' buttons to use `<Button>` component instead of `<TouchableHighlight>`. 

## Screenshots

No visual difference, just a change in the UIA tree to remove extra Text component which was having unintended Accessibility problems.

### CODE REVIEW

**Comment by chrisglein** [comment]
Both the linked issues are closed:
- https://github.com/microsoft/react-native-gallery/issues/302
- https://github.com/microsoft/react-native-gallery/issues/307
(from your PR 2 weeks ago)

What's the addition here?

**Comment by YajurG** [comment]
> What's the addition here?

Made some changes addressing feedback from the accessibility team in order to get their sign off. This is the most up to date version of the package that we sent to them yesterday.

File: windows/rngallery/Package.appxmanifest [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
  <Identity
     Name="Microsoft.ReactNativeGallery"
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
-    Version="1.0.18.0" />
+    Publisher="CN=Microsoft Corporation (Internal Use Only), O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
```
Comment (AgneLukoseviciute) [inline windows/rngallery/Package.appxmanifest:16]: I don't think we want to merge this change into the repo, only needed as a local change when signing a new package through the process you used.

File: windows/rngallery/rngallery.vcxproj [hunk 2]
```diff
@@ -49,6 +49,15 @@
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <CharacterSet>Unicode</CharacterSet>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
```
Comment (AgneLukoseviciute) [inline windows/rngallery/rngallery.vcxproj:52]: same for this, don't think we want this merged into the repo.

**Review by AgneLukoseviciute — COMMENTED** [review]
This seems to be missing the SketchCanvas changes to fix the lack of UIA control types: 
image

was that included in the original SketchCanvas PR?

**Comment by YajurG** [comment]
> This seems to be missing the SketchCanvas changes to fix the lack of UIA control types

I think those changes have already been pushed upstream. I ran the build locally before pushing and the sketch page was updated with the UIA types as button. Did you run into this just now?

**Review by AgneLukoseviciute — APPROVED** [review]

**Pull request merged**